service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/Order.java

```diff
@@ -45,6 +45,7 @@
 import static uk.gov.hmcts.reform.fpl.model.order.OrderQuestionBlock.TRANSLATION_REQUIREMENTS;
 import static uk.gov.hmcts.reform.fpl.model.order.OrderQuestionBlock.UPLOAD_AMENDED_ORDER;
 import static uk.gov.hmcts.reform.fpl.model.order.OrderQuestionBlock.UPLOAD_ORDER_FILE;
+import static uk.gov.hmcts.reform.fpl.model.order.OrderQuestionBlock.VARY_OR_EXTEND_SUPERVISION_ORDER;
```

Comment:
this will be removed as the OrderQuestionBlock.* will be used for imports rather than the individual question blocks